plugins/Login/stylesheets/login.less

```
@@ -108,3 +108,82 @@
         color: gray;
     }
 }
+
+.onBoarding {
```

It's one word, so no capitalisation required in the middle.

```suggestion
.onboarding {
```

---

@michalkleiner I think its okay as IDE gives an error of typo in word when we change the camelCase to lowercase for this word

---

The IDE spell check isn't always the smartest one, it needs a bit of help sometimes.
In this case I'd suggest adding the word as a custom word if it finds it incorrect.

https://www.merriam-webster.com/dictionary/onboarding

---

@michalkleiner updated